src/pretix/control/forms/event.py

```diff
@@ -854,6 +854,15 @@ class DisplaySettingsForm(SettingsForm):
         label=_("Show variations of a product expanded by default"),
         required=False
     )
+    frontpage_subevent_ordering = forms.ChoiceField(
+        label=_('Subevent ordering'),
```

Comment:
Please replace by
```
pgettext('subevent', 'Date ordering')
```
since we don't expose the word "subevent" to end users :)

Reply:
Of course!